src/app/components/event/steps/Details.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div>
+    <h2 class="mb-4 text-xl">{{ t('eventDescription') }}</h2>
+    <div class="rounded-3xl border border-gray-100 bg-white p-4 shadow-sm">
+      <textarea
+        v-model="eventForm.description"
+        class="min-h-[400px] w-full resize-none border-0 text-lg text-gray-600 placeholder:text-gray-400 focus:outline-none"
+        :placeholder="t('eventDescriptionPlaceholder')"
+        maxlength="2500"
+        @input="onDetailsInput"
+      />
+    </div>
+    <div class="mt-2 text-right text-lg text-gray-600">
+      {{ eventForm.description.length }}{{ t('maxCharacters') }}
```

Comment:
```suggestion
      {{ t('maxCharacters', { count: eventForm.description.length }) }}
```

Then you can insert count as a variable in the translation string below and get language-dependent formatting (some languages may reverse the order of texts).